src/code_80005FD0.c

```diff
@@ -4950,7 +4949,7 @@ f32 func_80013C74(s16 arg0, s16 arg1) {
     return var_f2;
 }
 
-void func_80013D20(VehicleStuff *vehicle) {
+void update_vehicle_on_waypoint(VehicleStuff *vehicle) {
```

Comment:
vehicle_follow_waypoints

something kinda like that?

Reply:
I want to keep update to mean it's use in a update loop and it change data of vehicle

Reply:
That's fine. But something similar telling the reader that it's following the waypoints would be good.

Or... route-constrained or fixed path, or something like that.
set_vehicles_fixed_path ?

Reply:
(i.e. `on_waypoint` isn't quite right)

Reply:
if you see lastest commit I remane it to `update_vehicle_following_waypoint`